composer/helpers/src/DependabotInstallationManager.php

```diff
@@ -13,9 +13,9 @@
 
 class DependabotInstallationManager extends InstallationManager
 {
-    private $installed = [];
-    private $updated = [];
-    private $uninstalled = [];
+    private array $installed = [];
+    private array $updated = [];
+    private array $uninstalled = [];
```

Comment:
I'm not very familiar with PHP, but can't it figure out that this is an array by looking at the declaration? Is this annotation only for humans?

Reply:
@jurre 

It's not necessary to use property type declarations, but I would consider it best practice when using PHP 7.4 and newer!

💁‍♂️ For reference, see https://www.php.net/manual/en/migration74.new-features.php#migration74.new-features.core.typed-properties.